iib/workers/tasks/build_merge_index_image.py

```diff
@@ -250,6 +254,13 @@ def handle_merge_request(
                 overwrite_target_index_token,
             )
 
+        _add_label_to_index(
```

Comment:
For `Add` request, we call this function in the handle_add_request  because that's where we add the ocp_version label too. Can I please suggest that we move this call to `_add_bundles_missing_in_source` because that's where we add the ocp_version label. The other option is to move the `ocp_version` call here in this function. I would leave it up to you. The latter would be ideal but I would be okay if you move this too to keep things consistent. :)